crates/primitives/src/transaction/mod.rs

```diff
@@ -32,6 +33,10 @@ mod signature;
 mod tx_type;
 pub(crate) mod util;
 
+// Expected number of transactions where we can expect a speed-up by recovering the senders in
+// parallel.
+const PARALLEL_SENDER_RECOVERY_THRESHOLD: usize = 10;
```

Comment:
is this an estimation or was it benched?

Reply:
both - it was benched in my machine (it was already better at 5 transactions tbh), but each machine will have different results and 10 seems like a good number. probably there is a better way

Reply:
done, lmk